src/effects/RealtimeEffectState.cpp

```diff
@@ -213,20 +213,16 @@ struct RealtimeEffectState::Access final : EffectSettingsAccess {
    std::weak_ptr<RealtimeEffectState> mwState;
 };
 
-RealtimeEffectState::RealtimeEffectState(const PluginID & id)
+RealtimeEffectState::RealtimeEffectState(const PluginID& id)
 {
    SetID(id);
+   BuildAll();
 }
 
-RealtimeEffectState::RealtimeEffectState(const RealtimeEffectState &other)
```

Comment:
The commit that removes this copy constructor should also remove the comment in the header file, `Members that are copied for undo and redo`